lib/operations/command.rb

```diff
@@ -131,7 +131,20 @@ class Operations::Command
   include Dry::Equalizer(*COMPONENTS)
   extend Dry::Initializer
 
+  # Provides message and meaningful sentry context for failed operations
   class OperationFailed < StandardError
+    attr_reader :data
+
+    def initialize(data)
+      @data = data
+      operation_class_name = data.operation&.operation&.class&.name
+
+      super("#{operation_class_name} failed on #{data.component}")
+    end
+
+    def sentry_context
```

Comment:
I've got a feeling that we have to define this method as a monkeypatch in every application. Cause you now, maybe some of the apps are using something else? This doesn't belong to the gem. Though we can as well keep it.

Reply:
good point, it does not belong here, but we have sentry as a default config, so let's keep it here as a already provided solution for default logger, it should not break anything

Reply:
@pyromaniac I was about to comment the same, this is coupling to the external services, however... maybe we could just define this like a default method `#error_context` and in initialiser config allow providing a method name that would by default be aliased to `error_context`? Might sound a bit like an overhead but it provides cleaner configuration-based interface and we don't have to monkeypatch anything

Reply:
it sounds like an overhead, we can't even predict like other loggers will write errors